common/xbps-src/libexec/xbps-src-dopatch.sh

```diff
@@ -20,10 +20,6 @@ done
 
 setup_pkg "$PKGNAME" $XBPS_CROSS_BUILD
 
-for f in $XBPS_COMMONDIR/environment/patch/*.sh; do
```

Comment:
Presumably this was used for some package a long time ago and isn't now.  For my curiosity what was the use case here?

Reply:
If you look at the code, the files are sourced later on.

This was sourcing the files twice.

Reply:
if you look at the code, the files were sourced twice